### PR TITLE
Added livestream tutorial

### PR DESCRIPTION
**Pull request by emile-00**
Added the livestream metrics analysis tutorial. Written tutorial here: https://www.notion.so/risingwave-labs/Tutorial-Live-stream-metrics-analysis-f4e16582e3e249e99fb3019a051bff76

## Info
- **Description**: 
[ What's changed? ]

- **Related code PR**: 
[ Provide the link to the code PR here. For example, https://github.com/risingwavelabs/risingwave/pull/4085. Delete this part if there's no code PR related. ]

- **Related doc issue**: 
Resolves [ Provide the link to the doc issue here. ]

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>image</details>

## Published doc pages
  [ After merging this PR, edit the description to include the links to the updated doc pages here. For example, https://www.risingwave.dev/docs/latest/intro/. ]

### CODE REVIEW

**Comment by CharlieSYH** [comment]
I've updated `sidebar.js` to make this tutorial available on the sidebar. @emile-00

File: docs/tutorials/live-stream-metrics-analysis.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
+---
+id: live-stream-metrics-analysis
+slug: /live-stream-metrics-analysis
+title: Live stream metrics analysis
+description: RisingWave makes it possible to analyze live stream metrics in a low code manner.
+---
+
+## Overview
+
+Live streaming has become a popular form of entertainment where streamers can interact with their audience in real-time. Not only do streamers want to keep a high view count, but they also want to maintain a high-quality stream to ensure the viewing experience is enjoyable for their audience.
```
Comment (CharlieSYH) [inline docs/tutorials/live-stream-metrics-analysis.md]: According to Google Style Guide, 'real time' serves as a noun.
Therefore, in real-time -> in real time.

It's now in our style guide: https://www.notion.so/risingwave-labs/Word-usage-actively-updating-0af49cdd79e64d84a289561846c28265

File: docs/tutorials/live-stream-metrics-analysis.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+The first materialized view will summarize the streaming performance for each stream every 10 minutes. To create it, we will use the tumble function to map each event into a 10-minute window and aggregate based on each room to calculate how long the stream was frozen, the average number of packets lost per second, and the average round-trip time.
+
+```sql
+CREATE MATERIALIZED VIEW live_video_qos_10min AS
+SELECT
+    window_start AS report_ts,
+    room_id,
+    SUM(video_total_freeze_duration) AS video_total_freeze_duration,
+    AVG(video_lost_pps) as video_lost_pps,
```
Comment (CharlieSYH) [inline docs/tutorials/live-stream-metrics-analysis.md]: 'as' should be in all caps: 'AS'.

File: docs/tutorials/live-stream-metrics-analysis.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+    AVG(video_rtt) as video_rtt
```
Comment (CharlieSYH) [inline docs/tutorials/live-stream-metrics-analysis.md]: Same here~

File: docs/tutorials/live-stream-metrics-analysis.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+Next, we will set up two materialized views to keep track of view counts.
+
+The first materialized view will track the number of unique viewers on the entire streaming site every minute. We will use the tumble function to map each event into a one-minute window and count the number of distinct viewers within each time window.
+
+```sql
+-- A real-time dashboard of the total UV.
+CREATE MATERIALIZED VIEW total_user_visit_1min AS
+SELECT
+    window_start AS report_ts,
+    COUNT(DISTINCT user_id) as uv
```
Comment (CharlieSYH) [inline docs/tutorials/live-stream-metrics-analysis.md]: And here.

File: docs/tutorials/live-stream-metrics-analysis.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+2022-10-19 11:39:00	| 2
+```
+
+The second materialized view will track each streamer’s unique viewers every minute. We will use the tumble function to map each event into a one-minute window and then group by the room_id to each streamer’s unique viewers.
+
+```sql
+CREATE MATERIALIZED VIEW room_user_visit_1min AS
+SELECT
+    window_start AS report_ts,
+    COUNT(DISTINCT user_id) as uv,
```
Comment (CharlieSYH) [inline docs/tutorials/live-stream-metrics-analysis.md]: as -> AS

**Review by CharlieSYH — COMMENTED** [review]
Left a few comments. Please take a look, @emile-00. The rest looks gook!

File: docs/tutorials/live-stream-metrics-analysis.md (hunk 1 above)
Comment (emile-00) [reply docs/tutorials/live-stream-metrics-analysis.md]: Thanks for catching these details!

**Review by CharlieSYH — APPROVED** [review]

**Review by hengm3467 — APPROVED** [review]
Looks good to me

**Pull request merged**